src/rules/color-named/index.js

```diff
@@ -55,6 +56,11 @@ export default function (expectation, options) {
       valueParser(decl.value).walk(node => {
         const { value, type, sourceIndex } = node
 
+        if (
+          optionsMatches(options, "ignore", "inside-function")
+          && type === "function"
+          && !colorFunctionNames.has(value.toLowerCase())) { return false }
```

Comment:
I'm still not sure I get it exactly. It looks like you're *not* going to check colors inside non-color functions, e.g. `map-get`, but you *are* going to check color inside color functions? Why are you going to check colors inside color functions? Shouldn't those also be ignored?